src/sass/mui/_select.scss

```diff
@@ -117,4 +117,16 @@ $xFormLabelLineHeight: floor($mui-label-font-size * 1.25);
       background-color: mui-color('grey', '200');
     }
   }
+
+  > .optgroup-option {
+    text-indent: 1em;
+  }
+
+  > .optgroup-label {
+    color: #888888;
+
+    &:hover {
+      background-color: #ffffff;
```

Comment:
Is there a way to avoid canceling the hover effect by setting the background color?